os_site/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "texastribune-opensource",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch:sass": "sass --watch src/sass:_site/css --load-path=node_modules",
+    "build:sass": "sass src/sass:_site/css --load-path=node_modules",
+    "watch:eleventy": "eleventy --serve",
+    "build:eleventy": "eleventy",
+    "start": "npm-run-all build:sass --parallel watch:*",
+    "build": "npm-run-all build:*",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@11ty/eleventy": "^0.11.1",
+    "luxon": "^1.26.0",
+    "npm-run-all": "^4.1.5",
+    "react": "^17.0.1",
+    "sass": "^1.32.5",
+    "sass-mq": "^5.0.1"
+  },
+  "devDependencies": {
+    "@11ty/eleventy-navigation": "^0.1.6",
+    "@texastribune/queso-ui": "^9.6.0",
+    "typewriter-effect": "^2.17.0"
+  }
```

Comment:
This is more of a final step, but just adding a reminder to go back and check/remove any dependencies/devDependencies you don't end up using. I do this too and often forget to go back and clean up.